app/src/main/res/values/strings.xml

```diff
@@ -526,7 +526,17 @@ Upload your first media by tapping on the add button.</string>
   <string name="image_chooser_title">Choose Images to upload</string>
 
   <string name="please_wait">Please wait…</string>
+
+  <string name="nominating_for_deletion">Nominating for Deletion</string>
+  <string name="success">Success</string>
+  <string name="successfully_nominated">Successfully nominated</string>
+  <string name="for_deletion">for deletion</string>
+  <string name="failed">Failed</string>
+  <string name="could_not_request_deletion">Could not request deletion</string>
```

Comment:
For instance this should be `Could not request deletion.`

Reply:
Is it okay now